### ISSUE TITLE
Follow up to #262 CI Build-Artefacts

### ISSUE THREAD

**Issue by security-companion** [w4af/w4af#336]
Hi,
@codders Thank you for working on the circle-ci removal.
I found in w4af\plugins\tests\helper.py that there's an output_dir-variable generated that gets it's content from a circle-environment variable. Perhaps we can use github actions to store outputs when a build fails to get some deeper insight why it failed.
see eg. [link to github docu](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#about-workflow-artifacts)

**Comment by codders** [comment]
Hi @security-companion ,

That's a nice find! I didn't find that previously, and I can see how that might be useful. So far in my development experience I have always been able to reproduce issues locally if they showed up on CI, so I haven't had the need to dig through CI debug logs.

We can for sure think about adding such a functionality for the github actions. I would just take care not to run the CI jobs with DEBUG enabled by default, as that would likely slow the builds down.

**Comment by security-companion** [comment]
Thanks for your feedback. Are you sure that you need to activate debug by default for using the action [upload-artifact](https://github.com/actions/upload-artifact) described in the link above?

**Comment by codders** [comment]
No, not to use the upload-artifact. But if I understood the code in `helper.py`, the artifact that we upload is the debug log from the test runs, so there's no sense doing the artifact upload if the `DEBUG` flag is not enabled - there would be nothing to upload.
